packages/directory/tests/dummy/app/routes/reports/new.js

```diff
@@ -0,0 +1,6 @@
+import Route from '@ember/routing/route';
+import { inject } from '@ember/service';
+
+export default Route.extend({
```

Comment:
is this in the dummy app?

Reply:
Yes, when you create a new report from the directory, it will link here